AdyenDropIn/Components/PaymentMethodListComponent.swift

```diff
@@ -9,13 +9,48 @@ import Foundation
 import UIKit
 
 /// :nodoc:
-internal protocol LoadingComponent: PresentableComponent {
+public protocol LoadingComponent {
     /// :nodoc:
     func startLoading(for component: PaymentComponent)
+
+    /// Stops any processing animation that the view controller is running.
+    ///
+    /// - Parameters:
+    ///   - success: Boolean indicating the component should go to a success or failure state.
+    ///   - completion: Completion block to be called when animations are finished.
+    func stopLoading(withSuccess success: Bool, completion: (() -> Void)?)
+}
+
+extension LoadingComponent {
+
+    /// Stops any processing animation that the view controller is running.
+    func stopLoading() {
+        stopLoading(withSuccess: true, completion: nil)
+    }
+
+    /// Stops any processing animation that the view controller is running.
+    ///
+    /// - Parameters:
+    ///   - success: Boolean indicating the component should go to a success or failure state.
+    func stopLoading(withSuccess success: Bool) {
+        stopLoading(withSuccess: success, completion: nil)
+    }
+
+    /// Stops any processing animation that the view controller is running.
+    ///
+    /// - Parameters:
+    ///   - success: Boolean indicating the component should go to a success or failure state.
+    ///   - completion: Completion block to be called when animations are finished.
+    public func stopLoading(withSuccess success: Bool, completion: (() -> Void)?) {
+        completion?()
+    }
+
+    /// :nodoc:
+    public func startLoading(for component: PaymentComponent) { /* Empty default implementation */ }
 }
 
 /// A component that presents a list of items for each payment method with a component.
-internal final class PaymentMethodListComponent: LoadingComponent, Localizable {
+internal final class PaymentMethodListComponent: LoadingComponent, PresentableComponent, Localizable {
```

Comment:
Isn't any `PresentableComponent` also a `Localizable` ?
Except for ApplePay maybe..